### ISSUE TITLE
Constrast is insufficient for various texts

### ISSUE THREAD

**Issue by dessalines** [LemmyNet/jerboa#1503]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

The contrast between foreground and background is not sufficient for several texts: 

- #7e7a7a on #2a2526,  
- #80cbc4 on #f1e7e8,  
- #8e8a8a on #fff8f8
